### PR TITLE
Jc/fix jvm issue

### LINKED ISSUE #34: bug: classpath added to intellij runConfig VM Options
seems that the classpath and other values are being added to the VM Options of the intelliJ run configuration.

this may also be happening in eclipse.
```
-Dfile.encoding=UTF-8
-Duser.country=US
-Duser.language=en
-Duser.variant
-cp
/Users/<username>/temp/test-launch-config/test-launch-config-server/build/classes/main

/Users/<username>/temp/test-launch-config/test-launch-config-server/build/resources/main

/Users/<username>/temp/test-launch-config/test-launch-config-server/build/web-resources

/Users/<username>/temp/test-launch-config/test-launch-config-api/build/libs/test-launch-config-api-unspecified.jar

/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.config.crypto/encrypted-config-value-bundle-dropwizard1/0.4.0/21d6b354a2bafb00df010a27fdbc3b9743e49c9e/encrypted-config-value-bundle-dropwizard1-0.4.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.indexpage/dropwizard-index-page/1.0.0/e999bdd9f7c3fd8330ffd28832721807d7f068ff/dropwizard-index-page-1.0.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.multipass/dropwizard-multipass/1.24.0/e8c2f7ee954aa48d3f33781c643e23eefa6fed1f/dropwizard-multipass-1.24.0.jar
/Users/<username>/.gradle/caches/modules-2/file<username>s-2.1/com.palantir.remoting/http-servers/0.14.0/f605a0647d83c2751a0bde3a5234bc387367144f/http-servers-0.14.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.websecurity/dropwizard-web-security/1.0.0/44a80314a2dc43853b959888203c04202d477cb6/dropwizard-web-security-1.0.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/io.dropwizard/dropwizard-core/1.0.0/7a57ea6f91cd22ba02b8973c1a4e0f9f4caebd53/dropwizard-core-1.0.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/io.dropwizard-bundles/dropwizard-configurable-assets-bundle/1.0.0/c364ad283f1917d99d086d11f8e876d7b4300162/dropwizard-configurable-assets-bundle-1.0.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-databind/2.7.6/c7012a59b4f36843489753a2027e169c3b8586f8/jackson-databind-2.7.6.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/javax.ws.rs/javax.ws.rs-api/2.0.1/104e9c2b5583cfcfeac0402316221648d6d8ea6b/javax.ws.rs-api-2.0.1.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/io.dropwizard.metrics/metrics-annotation/3.1.2/7ee8e9afe8e15f921a3bc9f8d4b1109b58b63d46/metrics-annotation-3.1.2.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.config.crypto/encrypted-config-value-module/0.4.0/39bfa22ffe17f2031063d6628f6d1f7a9de133f4/encrypted-config-value-module-0.4.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.github.spullara.mustache.java/compiler/0.9.1/14aec5344639782ee76441401b773946c65eb2b3/compiler-0.9.1.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.google.code.findbugs/annotations/3.0.1/fc019a2216218990d64dfe756e7aa20f0069dea2/annotations-3.0.1.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.multipass/multipass-client/1.24.0/be817097e7394d5ba63de0ffd55696b01b2c5ac0/multipass-client-1.24.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.remoting/http-clients/0.9.0/87032f1ae342b727e4ae9baef7e0befb4b841493/http-clients-0.9.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.remoting/ssl-config/0.9.0/f96d2acc92be1efb2d2234ea43f972b386635cc5/ssl-config-0.9.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.remoting/error-handling/0.14.0/64b1e4177cc8af6399f77f9e2d64e349d20725bc/error-handling-0.14.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.remoting/brave-extensions/0.14.0/89cf13dc9abe5a97e7d8ea98d3951eea7b80ba23/brave-extensions-0.14.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/io.zipkin.brave/brave-jaxrs2/3.9.0/a2d165ce2dabd8162f96675c62a31930aa04fbd1/brave-jaxrs2-3.9.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/io.dropwizard/dropwizard-servlets/1.0.0/2069239ec3788f99d857fbfbb397cb18d6767e45/dropwizard-servlets-1.0.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/org.apache.commons/commons-lang3/3.4/5fe28b9518e58819180a43a850fbc0dd24b7c050/commons-lang3-3.4.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.config.crypto/encrypted-config-value/0.4.0/eaa624538211b3601d3fe588a5e3d06a0e5cc848/encrypted-config-value-0.4.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/net.jcip/jcip-annotations/1.0/afba4942caaeaf46aab0b976afd57cc7c181467e/jcip-annotations-1.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.multipass/multipass-api/1.24.0/f5388ec0b6229369e11b6f46776612ee29054e1f/multipass-api-1.24.0.jar
/Users/<username>/.gradle/caches/modules-2/files-2.1/com.palantir.multipass/multipass-token/1.24.0/853eb4ade0818f02c297b668cc14b94056be42dc/multipass-token-1.24.0.jar
...
```

### PR DESCRIPTION
**Pull request by jmcampanini**
fixes #34

### CODE REVIEW

**Comment by jmcampanini** [comment]
@qinfchen for CR

**Comment by jmcampanini** [comment]
i believe the tests are a bit incorrect. will modify

**Comment by jmcampanini** [comment]
there. this should fix it for intellij

File: src/main/groovy/com/palantir/launchconfig/IdeaLaunchConfigTask.groovy [hunk 1]
```diff
@@ -76,6 +76,16 @@ class IdeaLaunchConfigTask extends DefaultTask {
         ])
     }
 
+    /**
+     * Build the VM Params from the JVM Args and the System Properties.
+     */
+    private List<String> buildVmParams(JavaExec javaExec) {
+        List<String> vmParams = new ArrayList<String>()
+        vmParams.addAll(javaExec.jvmArgs)
```
Comment (qinfchen) [inline src/main/groovy/com/palantir/launchconfig/IdeaLaunchConfigTask.groovy]: how

File: src/main/groovy/com/palantir/launchconfig/IdeaLaunchConfigTask.groovy [hunk 2]
```diff
@@ -76,6 +76,16 @@ class IdeaLaunchConfigTask extends DefaultTask {
         ])
     }
 
+    /**
+     * Build the VM Params from the JVM Args and the System Properties.
+     */
+    private List<String> buildVmParams(JavaExec javaExec) {
```
Comment (qinfchen) [inline src/main/groovy/com/palantir/launchconfig/IdeaLaunchConfigTask.groovy]: buildJvmParams?

Comment (qinfchen) [inline src/main/groovy/com/palantir/launchconfig/IdeaLaunchConfigTask.groovy]: @jmcampanini ?

Comment (jmcampanini) [inline src/main/groovy/com/palantir/launchconfig/IdeaLaunchConfigTask.groovy]: we're building the values for the `VM_PARAMETERS value, figured it should match.

Comment (qinfchen) [inline src/main/groovy/com/palantir/launchconfig/IdeaLaunchConfigTask.groovy]: isn't all of values part of allJvmArgs?

Comment (qinfchen) [inline src/main/groovy/com/palantir/launchconfig/IdeaLaunchConfigTask.groovy]: your call though. just want flag that

**Pull request merged**